languages/vinyl.pot

```diff
@@ -32,39 +33,44 @@ msgid "An advanced WordPress audio player."
 msgstr ""
 
 #: vinyl.php
-#. plugin author
+#. Author of the plugin
 msgid "codekraft, bitmachina"
 msgstr ""
 
-#: vinyl.php:27
+#: vinyl.php
```

Comment:
@erikyo I notice the line numbers are missing here. Is that intentional?

Reply:
only for plugin header in the main plugin file, since it's parsed differently before the main extraction process. 

i need to do in that way to collect the required metadata :) anyhow the translation apps like poedit will point at the top of the file so, i believe the line number isn't required in this case

Reply:
Cool, thanks for the confirmation.